tests/test_output_render.py

```diff
@@ -103,22 +104,22 @@ def test_jjm_default_plan_output(test_case: JCase):
 
         # check default output
         result = runner.invoke(jjm, ["plan", "--skip-pager"], catch_exceptions=False)
-        print(result.output)
+        # print(result.output)
         assert result.output == test_case.output_default
         return_code = 0 if test_case.output_default == "No changes.\n" else 2
         assert result.exit_code == return_code
 
         # @TODO for use with testing #9 Fix
-        # # check yaml output
-        # result = runner.invoke(jjm, ["plan", "--output=yaml"], catch_exceptions=False)
-        # print(result.output)
-        # assert yaml.safe_load(result.output) == test_case.output_default
-        # return_code = 0 if test_case.output_struct == [] else 2
-        # assert result.exit_code == return_code
-        #
-        # # check json output
-        # result = runner.invoke(jjm, ["plan", "--output=json"], catch_exceptions=False)
-        # print(result.output)
-        # assert json.loads(result.output) == test_case.output_struct
-        # return_code = 0 if test_case.output_struct == [] else 2
-        # assert result.exit_code == return_code
+        # check yaml output
+        result = runner.invoke(jjm, ["plan", "--output=yaml"], catch_exceptions=False)
+        print(result.output)
+        assert yaml.safe_load(yaml.dump(result.output)) == test_case.output_yaml
```

Comment:
nah. You want to check if the structure from the output matches expectations.

To make this approach work you'd have to ensure pyyaml dumps out in the exact same format, likely tweaking  `default_flow_style=False` and `indent=`.

We'd be better off making sure:
 - the yaml is parsable
 - the parsed structure we get matches expectations